<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="12709" started-at="2016-04-14T19:53:49Z" finished-at="2016-04-14T19:54:02Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="12709" started-at="2016-04-14T19:53:49Z" finished-at="2016-04-14T19:54:02Z">
      <class name="com.globant.training.rocio_bossio.tests.Tests">
        <test-method status="PASS" signature="before()[pri:0, instance:com.globant.training.rocio_bossio.tests.Tests@6e1ec318]" name="before" is-config="true" duration-ms="6345" started-at="2016-04-14T16:53:49Z" finished-at="2016-04-14T16:53:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method status="PASS" signature="case1()[pri:0, instance:com.globant.training.rocio_bossio.tests.Tests@6e1ec318]" name="case1" duration-ms="25" started-at="2016-04-14T16:53:55Z" description="Verifying if title displayed is the expected one" finished-at="2016-04-14T16:53:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- case1 -->
        <test-method status="PASS" signature="after()[pri:0, instance:com.globant.training.rocio_bossio.tests.Tests@6e1ec318]" name="after" is-config="true" duration-ms="592" started-at="2016-04-14T16:53:55Z" finished-at="2016-04-14T16:53:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- after -->
        <test-method status="PASS" signature="before()[pri:0, instance:com.globant.training.rocio_bossio.tests.Tests@6e1ec318]" name="before" is-config="true" duration-ms="5265" started-at="2016-04-14T16:53:56Z" finished-at="2016-04-14T16:54:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method status="FAIL" signature="case3()[pri:0, instance:com.globant.training.rocio_bossio.tests.Tests@6e1ec318]" name="case3" duration-ms="3" started-at="2016-04-14T16:54:01Z" description="Verifying if a post date is the expected one" finished-at="2016-04-14T16:54:01Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
	at com.sun.proxy.$Proxy7.click(Unknown Source)
	at com.globant.training.rocio_bossio.pages.HomePage.goToSinglePostPage(HomePage.java:55)
	at com.globant.training.rocio_bossio.tests.Tests.case3(Tests.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- case3 -->
        <test-method status="PASS" signature="after()[pri:0, instance:com.globant.training.rocio_bossio.tests.Tests@6e1ec318]" name="after" is-config="true" duration-ms="449" started-at="2016-04-14T16:54:01Z" finished-at="2016-04-14T16:54:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- after -->
      </class> <!-- com.globant.training.rocio_bossio.tests.Tests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
